pycheribuild/config/chericonfig.py

```diff
@@ -328,6 +328,17 @@ def __init__(self, loader, action_class: "type[CheribuildActionEnum]") -> None:
             group=loader.cross_compile_options_group,
             help="The floating point/SIMD mode to use for building AArch64 programs",
         )
+
+        self.riscv_baremetal_hardfloat = loader.add_bool_option("riscv-baremetal-hardfloat", default=False,
+            group=loader.cross_compile_options_group,
+            help="Use hard floating point ABI for building CHERI-RISC-V programs")
+        self.riscv_cheri_rvc = loader.add_bool_option("riscv-cheri-rvc", default=False,
```

Comment:
C is the default behaviour, do you need this diff?

Reply:
@arichardson Not anymore, I added this option when it wasn't the default. I have deleted this diff now.